backend/db/sqliteschema/migrations.go

```diff
@@ -219,4 +219,66 @@ var migrations = []string{
 		-- The balance in satoshis
 		balance INTEGER DEFAULT 0
 	);`,
+
+	// Stores sites that user has manually added
+	`CREATE TABLE sites (
+		-- Site unique identification. The hostname of the site with protocol https://example.com
+		hostname TEXT PRIMARY KEY,
+		-- The role we play in the site ROLE_UNSPECIFIED = 0 | OWNER = 1 | EDITOR = 2
+		role INT NOT NULL DEFAULT 0,
+		-- Address of the LND node backing up this wallet. In case lndhub, this will be the 
+		-- P2P addresses to connect to that site in the format of multiaddresses. Space separated.
+		addresses TEXT NOT NULL,
+		-- The account ID of the site. We need a previous connection to the site so the 
+		-- actual account is inserted in the accounts table when handshake.
+		account_id INTEGER REFERENCES accounts ON DELETE CASCADE NOT NULL
+	) WITHOUT ROWID;`,
+
+	// Table that stores all the tokens not yet redeemed inside a site. Although this table is relevant only
+	// for sites at the beginning, keep in mind that any regular node can be upgraded to a site.
+	`CREATE TABLE unredeemed_invite_links (
+		-- Unique token identification. Random 8 char words
+		token TEXT PRIMARY KEY,
+		-- The role the token will allow ROLE_UNSPECIFIED = 0 OWNER = 1 EDITOR = 2
+		role INT NOT NULL DEFAULT 2,
+		-- Timestamp since the token will no longer be eligible to be redeemed. Seconds since  Jan 1, 1970
+		expiration_time INTEGER NOT NULL CHECK (expiration_time > 0)
+	) WITHOUT ROWID;`,
+
+	// Table that stores the role each account has inside a site. Although this table is relevant only
+	// for sites at the beginning, keep in mind that any regular node can be upgraded to a site.
+	`CREATE TABLE redeemed_invite_links (
+		-- The account id that has been linked to a role on this site
+		account_id INTEGER REFERENCES accounts ON DELETE CASCADE NOT NULL PRIMARY KEY,
+		-- The role the account holds ROLE_UNSPECIFIED = 0 | OWNER = 1 | EDITOR = 2
+		role INT NOT NULL
+	) WITHOUT ROWID;`,
+
+	// Stores all the records published on this site. Although this table is relevant only
+	// for sites at the beginning, keep in mind that any regular node can be upgraded to a site.
+	`CREATE TABLE web_publication_records (
+		-- Unique identifier
+		id INTEGER PRIMARY KEY,
+		-- doc id of the base document published. Not its references.
+		document_id INTEGER REFERENCES ipfs_blocks ON DELETE CASCADE NOT NULL,
+		-- doc version of the base document published. Not its references.
+		document_version TEXT NOT NULL,
+		-- Path this publication is published. If NULL then its not pinned. If / is root document.
+		path TEXT
+		
+	);`,
```

Comment:
I think here we should have some sort of UNIQUE constraint, but I'm not quite sure what combination of what would be useful. At least to convey that there can only be one record at any given path, so maybe at least path should have UNIQUE constraint (NULLs won't match against it, so it's fine to have it).

Reply:
>You're right